### ISSUE TITLE
Wanted! Otto and Falconcrest

### ISSUE THREAD

**Issue by Naigio** [elysium-project/classic-bug-tracker#873]
Lord Falconcrest (https://classicdb.ch/?npc=2597#drop) won't seemingly spawn in his location within stromgarde keep, which makes it hard to do (https://classicdb.ch/?quest=685) Wanted! Otto and Falconcrest
Nighthaven pvp (patch 1.9) (v1.12.1)

**Comment by Thenari** [comment]
Can you go ahead and put a in game ticket in. Red "?" Left of your bags. Under quest mobs. Ask a GM to find its spawn point and check if its bugged out. this kind of thing would likely just take time to respawn or if its unable to spawn a server reset or patch would likely kick it back into place.

**Comment by Thenari** [comment]
I'm pretty sure i can close this.

How long did you wait for him to respawn? He also has some conversation triggers with a patrol you can see in the chat.  /tar Lord Falconcrest should also help find him.

**Comment by Naigio** [comment]
Actually nevermind all that, you're probably right. Just close it

**Comment by Thenari** [comment]
GM watched him spawn but he walked into walls and went out of reach of players :/ it's on the to-do list

**Comment by RainElysium** [comment]
Fixed.

**Comment by Thenari** [comment]
Hello, Falconcrest and another non-quest elite next to him respawn very quickly (I'd say less than 10sec). The corpses also disappears on respawn, which likely cause wipe and no loots.
1st try:
We casted CC on Otto and Falconcrest and killed both casters beside him, but a caster respawned almost instantly. We worked around that keeping it in CC, killed Otto then Falconcrest killed us.
2nd try:
We CCed Falconcrest, Otto and the buggy mage, killed the other mage, then otto and than Falconcrest. One of us died so we were waiting to loot for him to come back, but Falconcrest respwned in no time and wiped us again.
